### PR TITLE
Fix azure fields names

### PR DESCRIPTION
**Pull request by narph**
Replaced  `type` with `provider` and `resource_group` with `group` in the azure resource object.

### CODE REVIEW

**Comment by narph** [comment]
jenkins test this

**Comment by narph** [comment]
jenkins test this

File: x-pack/filebeat/module/azure/activitylogs/_meta/fields.yml [hunk 1]
```diff
@@ -7,17 +7,38 @@
...
+        - name: claims
+          type: group
+          description: >
+            Authorization
+          fields:
+            - name: name
+              type: keyword
+              description: >
+                Initiated by user
+            - name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
```
[the remark this answers is not in the document]
Comment (narph) [reply x-pack/filebeat/module/azure/activitylogs/_meta/fields.yml]: @exekias , yes:
`azure.activitylogs.identity.claims.name`
`azure.activitylogs.identity.claims.http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name`
`azure.activitylogs.identity.claims.http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`
`azure.activitylogs.identity.claims.http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`

are used as filters for the dashboards.

An example of the filebeat index (when the activity has been initiated by an user):
```
....
 "claims" : {
                  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier" : "...",
                  "aio" : "...",
                  "iss" : "...",
                  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn" : ...",
                  "puid" : "...",
                  "wids" : "...",
                  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname" : "...",
                  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname" : "...",
                  "exp" : "1571242827",
                  "ipaddr" : "...",
                  "iat" : "...",
                  "ver" : "1.0",
                  "groups" : "...",
                  "http://schemas.microsoft.com/claims/authnmethodsreferences" : "pwd",
                  "uti" : "...",
                  "aud" : "https://management.core.windows.net/",
                  "http://schemas.microsoft.com/claims/authnclassreference" : "1",
                  "nbf" : "...",
                  "appidacr" : "2",
                  "http://schemas.microsoft.com/identity/claims/tenantid" : "...",
                  "appid" : "...",
                  "name" : "...",
                  "http://schemas.microsoft.com/identity/claims/objectidentifier" : "...",
                  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name" : "...",
                  "http://schemas.microsoft.com/identity/claims/scope" : "user_impersonation"
                }
....
```

[the remark this answers is not in the document]
Comment (exekias) [reply x-pack/filebeat/module/azure/activitylogs/_meta/fields.yml]: Oh understood, thanks. What about the rest of claims in that example? I see `aio`, `iss` and many more. Should we map those?

**Pull request merged**